src/PowerShell/Microsoft.WinGet.Client/ModuleFiles/Microsoft.WinGet.Client.psd1

```diff
@@ -100,7 +100,7 @@ CmdletsToExport = @(
 # VariablesToExport = @()
 
 # Aliases to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no aliases to export.
-AliasesToExport = @()
+    AliasesToExport  = @('awgs', 'awgpm', 'dwgs', 'ewgs', 'ewgp', 'fdwgp', 'gwgp', 'gwgse', 'gwgso', 'gwgus', 'gwgv', 'iswgp', 'rwgs', 'rpdwgpm', 'rswgs', 'swgus', 'twgus', 'uswgp', 'udwgp')
```

Comment:
Should this include the old names for the singularized cmdlets?

Reply:
Yes it should, will include in the next update